README.md

```diff
@@ -197,7 +197,7 @@ In there we see a few interesting things:
 
 > Note that the actual commit hashes will be different for you. If you want to know how exactly git arrives at those revision IDs have a look at [this interesting article ](https://blog.thoughtram.io/git/2014/11/18/the-anatomy-of-a-git-commit.html).
 
-To compare that commit and the one one before we can do `git diff <commit>^!`, where the `^!` tells git to compare to the commit one before. So in this case I run `git diff 87a4ad48d55e5280aa608cd79e8bce5e13f318dc^!`
+To compare that commit and the one one before we can do `git diff <commit>^! (where the `^!` tells git to compare to the commit one before) In this case I run `git diff 87a4ad48d55e5280aa608cd79e8bce5e13f318dc^!`
```

Comment:
Reads better, thank you :)

You've accidentally removed the backtick closing of the inline code block after `git diff <commit>^!` 
Please add the ``` ` ``` there again

Reply:
Sure thing. Thanks for letting me know 👍